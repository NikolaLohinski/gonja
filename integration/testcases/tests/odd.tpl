{{ 2 is odd }}
{{ 1 is odd }}
{{ 0 is odd }}
{{ 2.0 is odd }}
{{ 2.1 is odd }}
{{ 'test' is odd }}
