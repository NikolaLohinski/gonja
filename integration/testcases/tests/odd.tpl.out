False
True
False
False
False
False
